experimental/cmd/sn3218/main.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package main
+
+import (
+	"log"
+	"time"
+
+	"periph.io/x/periph/conn/i2c/i2creg"
+	"periph.io/x/periph/experimental/devices/sn3218"
+	"periph.io/x/periph/host"
+)
+
+func main() {
+
+	if _, err := host.Init(); err != nil {
+		log.Fatal(err)
+	}
+
+	b, err := i2creg.Open("")
+
+	defer b.Close()
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	d, err := sn3218.New(b)
+
+	defer d.Halt()
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	d.Enable()
```

Comment:
Still check for errors everywhere, even if tedious.

Reply:
Should be really done now